src/modules/comments-counter.js

```diff
@@ -0,0 +1,6 @@
+const countComments = (node) => {
+  const nodes = [...node.children];
+  return nodes.length;
+};
+
+export default countComments;
```

Comment:
Good job for showing the number of comments. Please also mention the number of comments on the Cards as well as on the Home Page as you can see in the screenshot. 👍🏻 

<img width="817" alt="Screenshot 2022-10-21 at 10 25 42 PM" src="https://user-images.githubusercontent.com/83048208/197253955-db8bbe46-b966-4659-8a8a-208b27ef2691.png">

Reply:
Thank you so much for the suggestion. We will try to implement that. However I feel like this should be an optional suggestion as it is not one of the requirements and each team member is allocated either the homepage or the popup window. Perhaps you can give us a hint on how to implement this 👉👈 :sweat_smile:  @zainsadaqat